go-controller/pkg/libovsdbops/qos.go

```diff
@@ -105,3 +105,23 @@ func RemoveQoSesFromLogicalSwitchOps(nbClient libovsdbclient.Client, ops []libov
 	modelClient := newModelClient(nbClient)
 	return modelClient.DeleteOps(ops, opModels)
 }
+
+func RemoveQoSsFromLogicalSwitchesWithPredicateOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation,
```

Comment:
nit: `Remove` vs `Delete`

Reply:
I think all methods that have `FromLogicalSwitch` in the name start with `Remove`, e.g. `RemoveLoadBalancersFromLogicalSwitchOps`, `RemoveQoSesFromLogicalSwitchOps`, `RemoveACLsFromLogicalSwitchesWithPredicate`. We can update them all together if there is some general rule behind that